common/app/views/fragments/nav/newHeaderMenu.scala.html

```diff
@@ -4,36 +4,35 @@
 @import conf.switches.Switches.SearchSwitch
 
 @sectionList(topLevelSection: NewNavigation.EditionalisedNavigationSection) = {
-    <li class="menu-item js-navigation-item"
-        data-section-name="@topLevelSection.name">
-        <details class="js-close-nav-list"
-                 id="primary-list-@topLevelSection.name"
-                 open>
-            <summary class="menu-item__title js-navigation-button"
-                     data-link-name="nav2 : secondary : @topLevelSection.name">
+    @defining(Edition(request)) { edition =>
+        <li class="menu-item js-navigation-item"
+            data-section-name="@topLevelSection.name">
+            <a href="@{topLevelSection.getPopularEditionalisedNavLinks(edition)(0).url}"
```

Comment:
[Same dilemma](https://github.com/guardian/frontend/pull/14548#issuecomment-251123272) in a new context... Also headlines is a hidden home 😆

I wonder if clickthroughs would suffer if we got rid of all these “home” links and just left the 2nd-level pillars actionable? As is it's a tad confusing to have

opinion
opinion
opinion home (effectively 3rd opinion)

don‘t you think?